excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -405,9 +405,9 @@ public void entityDoesNotExistInRegistry_createErrorMessageCalled_defaultMessage
     }
 
     @Test
-    public void caseTypeEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenIdamRoleMissing() {
+    public void caseTypeEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenIdamAPMissing() {
```

Comment:
REname test when validation error is renamed

Reply:
Fixed in 3951e2b